tmt/steps/report/reportportal.py

```diff
@@ -116,10 +120,8 @@ class ReportReportPortal(tmt.steps.report.ReportPlugin[ReportReportPortalData]):
 
         export TMT_PLUGIN_REPORT_REPORTPORTAL_${MY_OPTION}=${MY_VALUE}
 
-    The optional launch name doesn't have to be provided if it is the
-    same as the plan name (by default). Assuming the URL and token
-    are provided by the environment variables, the plan config can look
-    like this:
+    Assuming the URL and token are provided by the environment variables,
+    the plan config can look like this:
```

Comment:
Is the statement about the launch name no longer true? It seems to be still correct, judging by line 313, `launch_name = self.data.launch or self.step.plan.name`.

Reply:
It is true, but I find it unnecessary there, if it's just a few lines above mentioned as optional. And it is also mentioned in options' description, see `launch` and `suite-per-plan`. The original sentence could be confusing, as it uses plan summary of the first plan only with option --suite-per-plan.